ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -1771,16 +1781,20 @@ protected PutRequest createPutRequest() {
   }
 
   /**
-   * Return a string message that will be used in {@link ByteBuf#touch()};
+   * Return a string message that will be used as a context for logging;
    * @return
    */
-  private String makeTouchMessage() {
+  private String makeLoggingContext() {
     if (restRequest == null || restRequest.getRestRequestContext() == null) {
       return "";
     }
     String channelStr =
         ((NettyRequest.NettyRequestContext) restRequest.getRestRequestContext()).getChannel().toString();
-    return restRequest.getUri() + " " + channelStr;
+    if (isEncryptionEnabled) {
+      return "True " + channelStr;
```

Comment:
What would the return string looks like? 

I feel we can make it more informative and formatted.  

maybe: 
channelStr + " Encrypted: True"

Reply:
I added Encrypted to the context.